### ISSUE TITLE
Harvest should support poller names with spaces in their names

### ISSUE THREAD

**Issue by cgrinds** [NetApp/harvest#2773]

**Comment by ybizeul** [comment]
There might be other places where a replacement with _ takes place though, see :
![image](https://github.com/NetApp/harvest/assets/182035/f343e1e7-58d4-439f-aafd-a13a05e875cd)

**Comment by cgrinds** [comment]
Thanks @ybizeul, can you describe what you expect to see in the screenshot above? Are you saying that your cluster is named `Webscale Demo`?

**Comment by ybizeul** [comment]
Yes, that cluster name should be "Webscale Demo" with a space. That's what cluster topology API returns in `data.name`

**Comment by cgrinds** [comment]
Thanks for clarifying. Fixed with #2780 

<img width="458" alt="image" src="https://github.com/NetApp/harvest/assets/242252/e4148f15-603e-42c0-a4d8-de0dd8e2316b">
